Source/Scene/Cesium3DTilesetCache.js

```diff
@@ -0,0 +1,79 @@
+define([
+        '../Core/defined',
+        '../Core/DoublyLinkedList'
+    ], function(
+        defined,
+        DoublyLinkedList) {
+    'use strict';
+
+    /**
+     * Stores tiles with content loaded.
+     *
+     * @private
+     */
+    function Cesium3DTilesetCache() {
```

Comment:
I don't think it need to be unit tested alone since it depends pretty heavily on having a tileset. The cache related tests in `Cesium3DTileset` are pretty good and this file has 100% coverage.